apisix/cli/ngx_tpl.lua

```diff
@@ -626,12 +626,15 @@ http {
     {% end %}
 
     server {
+        {% if enable_http2 then %}
+        http2 on;
```

Comment:
The new `http2` directive from https://nginx.org/en/docs/http/ngx_http_v2_module.html